fvm/environment/account-key-metadata/errors.go

```diff
@@ -0,0 +1,63 @@
+package accountkeymetadata
+
+import (
+	"fmt"
+)
+
+type KeyMetadataMalfromedError struct {
```

Comment:
This is not a user error. We should make sure it is (or gets wrapped into) a `CodedFailure`. same with any others here